tools/meson-git-contrib.sh

```diff
@@ -4,4 +4,5 @@ git shortlog -s `git describe --abbrev=0`.. | \
         cut -c8- | \
         sed 's/ / /g' | \
         awk '{ print $$0 "," }' | \
+        sed -e 's/ / /g' | \
```

Comment:
the contrib list in the NEWS file looks really weird if this isn't applied I must say. Moreover, this didn't use to be done this way by git, it was a recent change, and not sure it really makes sense to do that for them..